cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Cloud/Infra VMs."""
+import pytest
+
+from cfme.cloud.provider import CloudProvider
+from cfme.common.vm_views import PolicySimulationDetailsView
+from cfme.common.vm_views import PolicySimulationView
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(2),
+    pytest.mark.provider(
+        classes=[CloudProvider, InfraProvider],
+        selector=ONE_PER_CATEGORY,
```

Comment:
I'm thinking we want larger collection pool for this test - that we should test simulation against each provider type.

Reply:
@mshriver Since this is a UI test, my thought is that the specific provider is not so important, only that we have one of each type.